verification.py

```diff
@@ -272,10 +272,25 @@ def f():
             self._run_mutation_stats(key, command_line)
         self._instances[key] = f
 
-    def _run_mutation_discoal_stats(self, key, args):
+    def _discoal_str_to_ms(self, args):
         # convert discoal string to msprime string
         tokens = args.split(" ")
-        msp_str = " ".join(tokens[0:2]+tokens[3:])
+        # cut out sites param
+        del(tokens[2])
```

Comment:
FYI, ``del`` is a statement not a function so the brackets are uncessary here.

Reply:
ugh sorry.